### PR TITLE
Added Rubocop Cop:Rails/Delegate

### LINKED ISSUE #1488: Enable Rails/Delegate Rubocop
Enable Rails/Delegate Rubocop cop
https://rubocop.readthedocs.io/en/latest/cops_rails/#railsdelegate

### PR DESCRIPTION
**Pull request by divyanshumehta**
Enabled Rubocop Rails/Delegate. Fixes #1488

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ AllCops:
 #################### Style ###############################
 
 # Align the elements of a hash literal if they span more than one line
-Style/AlignHash:
+Style/HashSyntax:
```
Comment (Ana06) [inline .rubocop.yml]: Why did you change this? : confused: they are different cops.

File: app/serializers/speaker_serializer.rb [hunk 2]
```diff
@@ -3,7 +3,5 @@ class SpeakerSerializer < ActiveModel::Serializer
 
   attributes :name, :affiliation, :biography
 
-  def name
-    object.name
-  end
+  delegate :name, to: :object
```
Comment (Ana06) [inline app/serializers/speaker_serializer.rb]: This should be in the first commit :wink:

File: .rubocop_todo.yml [hunk 3]
```diff
@@ -124,12 +124,6 @@ Rails/Blank:
 Rails/Date:
   Enabled: false
 
-# Offense count: 1
-# Cop supports --auto-correct.
-Rails/Delegate:
```
Comment (Ana06) [inline .rubocop_todo.yml]: This should be in the first commit. Only the addition of the link should be here. :wink:

Comment (divyanshumehta) [inline .rubocop_todo.yml]: oh yes I think I have messed up with the commits and files.

File: .rubocop.yml (hunk 1 above)
Comment (divyanshumehta) [inline .rubocop.yml]: Sorry that was a typo.

**Comment by divyanshumehta** [comment]
Mades the changes @Ana06

File: app/serializers/speaker_serializer.rb (hunk 2 above)
Comment (divyanshumehta) [inline app/serializers/speaker_serializer.rb]: Changed it
